infrastructure/prometheus/shop/prometheus.yml

```
@@ -0,0 +1,16 @@
+rule_files:
+  - "/etc/prometheus/rules/*.yml"
+
+scrape_configs:
+  - job_name: 'prometheus-shop'
+    static_configs:
+      - targets: [ 'localhost:9093' ]
+
+  - job_name: 'shop-service'
+    metrics_path: '/actuator/prometheus'
+    static_configs:
+      - targets: [ 'host.docker.localhost:10053' ]
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)